### DISCUSSION TITLE
How to change colour of messages tagged as "Important"

### DISCUSSION THREAD

**Discussion by TigerFox57** [the-djmaze/snappymail#1168] [Q&A]
I sometimes get messages that have been tagged as "Important". Since I use SnappyMail in dark mode, these messages show in a dark red font on a black background, and as I am colour blind I can barely see what they say. Is there some way to alter this colour to something more visible to me? In Thunderbird I managed to change it to bright yellow, which I can see. Thanks.

UPDATE: I edited the file app.min.css to have `.msgflag-\$label1:not(.focused){color:yellow` which has done what I wanted, although this will get overwritten when I update SnappyMail no doubt.

**Comment by the-djmaze** [comment]
I understand what you want. There are many color blinds with different aspects.
For each there should be a different theme with high contrasts.

The best is to write your own custom theme in `/themes/` folder and change the colors to best fit your disabilities.

Read https://github.com/the-djmaze/snappymail/wiki/Custom-Themes
